src/components/dev-dot-content/mdx-components/mdx-alert/docs.mdx

```diff
@@ -4,6 +4,8 @@ componentName: 'MdxInlineAlert'
 
 The `MdxInlineAlert` component is used in long-form MDX content to highlight various message types. Default titles and icons are provided for each 'type', however these can be overridden via the `title` prop.
 
+_Note_ Authors use specific variants of this component in content where the type is set: `Tip`, `Highlight`, `Note`, & `Warning`: see `./variants.tsx`
```

Comment:
I will write detailed docs in the tutorials repo on how exactly to use this component once we 100% land on the authoring interface.

Reply:
💭 Can we add the detailed docs here and then link to it where you're thinking of in `hashicorp/tutorials`? That'd make the information available to both tutorials and docs authors in a way that they share the source of truth.

Reply:
For sure! I was planning on linking the two just haven't written the tutorials docs yet.

Reply:
Okay I added some authoring docs with example mdx content. I'll open PRs to add links to this in the tutorials repo. I'll need to find out where docs authors reference the available components to use 🤔